packages/@lwc/engine-core/src/framework/restrictions.ts

```diff
@@ -353,34 +351,6 @@ function getLightningElementPrototypeRestrictionsDescriptors(
         }),
     };
 
-    forEach.call(getOwnPropertyNames(globalHTMLProperties), (propName: string) => {
-        if (propName in proto) {
-            return; // no need to redefine something that we are already exposing
-        }
-        descriptors[propName] = generateAccessorDescriptor({
```

Comment:
We don't have any test for this in this repository. Those restrictions don't provide much information IMHO. If you have a strong opinion I can add them back.

Reply:
I don't recall the motivation behind these restrictions. I'd feel more comfortable removing them if we made the tradeoff clear before agreeing to remove them. Should we discuss that in this PR or a separate one?

Reply:
I did rever than change, I will open a separate issue to discuss this.